src/Sulu/Bundle/WebsiteBundle/Analytics/AnalyticsManager.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Analytics;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sulu\Bundle\WebsiteBundle\Entity\Analytic;
+use Sulu\Bundle\WebsiteBundle\Entity\AnalyticRepository;
+use Sulu\Bundle\WebsiteBundle\Entity\Domain;
+use Sulu\Bundle\WebsiteBundle\Entity\DomainRepository;
+
+/**
+ * Manages analytics.
+ */
+class AnalyticsManager implements AnalyticsManagerInterface
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * @var AnalyticRepository
+     */
+    private $repository;
+
+    /**
+     * @var DomainRepository
+     */
+    private $domainRepository;
+
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        AnalyticRepository $repository,
```

Comment:
maybe i should also rename `AnalyticsManager` to `AnalyticManager` because of `ContactManager`